proto/v1/role_service.proto

```diff
@@ -108,4 +123,12 @@ message Role {
   string title = 2;
   string description = 3;
   repeated string permissions = 4;
+
+  enum Type {
+    TYPE_UNSPECIFIED = 0;
+    BUILD_IN = 1;
```

Comment:
```suggestion
    SYSTEM = 1;
```

Reply:
or BUILT_IN

Reply:
use BUILT_IN, consistent with built-in SQL review rules